src/modules/fancyzones/editor/FancyZonesEditor/Models/Settings.cs

```diff
@@ -418,24 +421,58 @@ private void ParseCommandLineArgs()
             WorkArea = SystemParameters.WorkArea;
 
             string[] args = Environment.GetCommandLineArgs();
-            if (args.Length == 6)
+
+            if (args.Length == 2)
+            {
+                if (args[1].Equals("Debug"))
+                {
+                    const string debugDeviceInfoJson =

```

Comment:
Why do we need this?

Reply:
Removed. "Debug" argument is still there if user tries to run Editor.exe manually, the appropriate error message is shown instead of entering our 'debug mode'